keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -227,10 +227,10 @@ legacy tokens for security practices.
 
 #### Beta -> GA Graduation
 
-- [ ] In use by multiple distributions
-- [ ] Approved by PRR and scalability
-- [ ] Any known bugs fixed
-- [ ] Tests passing
+- [x] In use by multiple distributions
```

Comment:
GKE is deploying 1.27 with this enabled, check with a couple other distributions to make sure we're actually satisfying this requirement

Reply:
/cc @ritazh for Microsoft and @deads2k for RedHat

by default this feature is enabled, do you know if this feature is ever disabled?